src/api/v1/utils.ts

```diff
@@ -7,21 +7,52 @@ import { ApiResponse } from './schema/response';
 
 
 /*
- * We only choose to re-format the boom response
+ * Boom error type does not include key details
+ * added by Joi failAction validation method
+ */
+export interface BoomWithValidation extends Boom {
```

Comment:
This can be `type ... = Boom<any> & Joi.ValidationError;` and it'll cover the other fields.